sys-apps/66-tools/66-tools-0.1.1.0.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Package name: sys-apps/66-tools
+DESCRIPTION="Set of helper tools for execline and 66"
+HOMEPAGE="https://web.obarun.org/software/66-tools/0.1.1.0/index/"
+SRC_URI="https://git.obarun.org/Obarun/${PN}/-/archive/${PV}/${P}.tar.gz"
+
+LICENSE="0BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="static static-libs elogind dbus"
+
+DEPEND=" >=dev-libs/skalibs-2.14.3.0 >=sys-libs/oblibs-0.3.1.0
+dbus? (
+ elogind? ( sys-auth/elogind )
+ !elogind? ( sys-libs/basu )
+)
+"
+
+RDEPEND=" >=dev-lang/execline-2.9.6.1
+!static? ( ${DEPEND} )
+"
+# dbus? ( sys-apps/dbus-broker[-launcher] ) has been omitted for now
+# It mangles dependencies; Causes issues on musl etc..
+# The frontend will also be included in this package soon.
+
+BDEPEND="app-text/lowdown"
+
+pkg_setup() {
+if use dbus; then
```

Comment:
Consider [`optfeature.eclass`](https://devmanual.gentoo.org/eclass-reference/optfeature.eclass/index.html). This message should be output in postinst, possibly guarded by `REPLACING_VERSIONS`